media-libs/mesa/mesa-13.0.0.ebuild

```diff
@@ -44,11 +44,11 @@ done
 
 IUSE="${IUSE_VIDEO_CARDS}
 	bindist +classic d3d9 debug +dri3 +egl +gallium +gbm gcrypt gles1 gles2
-	+llvm nettle +nptl opencl osmesa pax_kernel openmax +openssl pic selinux
-	vaapi valgrind vdpau wayland xvmc xa kernel_FreeBSD"
+	libressl +llvm nettle +nptl opencl osmesa pax_kernel openmax +openssl pic
+	selinux vaapi valgrind vdpau wayland xvmc xa kernel_FreeBSD"
 
 REQUIRED_USE="
-	|| ( gcrypt nettle openssl )
+	|| ( gcrypt nettle openssl libressl )
```

Comment:
remove libressl here again. `USE="libressl"` does not unconditionally request Libressl.